Tutorials/preflight.xml

```diff
@@ -0,0 +1,1056 @@
+                    <value>0.4</value>
+                </greater-than>
+            </condition>
+        </error>
+        <exit>
+            <condition>
+                <less-than>
+                    <property>/sim/model/door-positions/leftDoor/position-norm</property>
+                    <value>0.4</value>
+                </less-than>
+            </condition>
+        </exit>
+    </step>
+
+    <step>
+        <message>Remove the pitot tube cover.</message>
+        <view>
+            <heading-offset-deg>140.0</heading-offset-deg>
+            <pitch-offset-deg>3.9</pitch-offset-deg>
+            <roll-offset-deg>0.0</roll-offset-deg>
+            <x-offset-m>-0.9</x-offset-m>
+            <y-offset-m>0.4</y-offset-m>
+            <z-offset-m>-1.0</z-offset-m>
+            <field-of-view>85.0</field-of-view>
+        </view>
+        <marker>
+            <x-m>-0.2000</x-m>
+            <y-m>-1.5190</y-m>
+            <z-m>0.4471</z-m>
+            <scale>3.1983</scale>
+        </marker>
+        <error>
+            <message>To enable the Airspeed Indicator to show something, you have to remove the pitot tube cover.</message>
+            <condition>
+                <property>sim/model/c172p/pitot-cover</property>
+            </condition>
+        </error>
+        <exit>
+            <condition>
+                <not>
+                    <property>sim/model/c172p/pitot-cover</property>
+                </not>
+            </condition>
+        </exit>
+    </step>
+
+    <step>
+        <message>You have to check if the fuel is contaminated with water. Take a fuel sample.</message>
+        <view>
+            <heading-offset-deg>140.0</heading-offset-deg>
+            <pitch-offset-deg>3.9</pitch-offset-deg>
+            <roll-offset-deg>0.0</roll-offset-deg>
+            <x-offset-m>-0.6</x-offset-m>
+            <y-offset-m>0.4</y-offset-m>
+            <z-offset-m>-1.0</z-offset-m>
+            <field-of-view>85.0</field-of-view>
+        </view>
+        <nasal>
+            <script>fgcommand("dialog-show", props.Node.new({"dialog-name": "c172p-left-fuel-sample-dialog"}));</script>
+        </nasal>
+        <error>
+            <message>To prevent the engine from failing the fuel must not contain any water. You have to take a sample of the fuel.</message>
+            <condition>
+                <not>
+                    <property>/consumables/fuel/tank[0]/fuel-sample-taken</property>
+                </not>
+            </condition>
+        </error>
+        <exit>
+            <condition>            
+                <property>/consumables/fuel/tank[0]/fuel-sample-taken</property>
+            </condition>
+        </exit>
+    </step>
+
+    <step>
+        <message>If the color of the fuel is light blue, the fuel isn't contaminated and you can return the sample to the tank. If it's partially translucid, it is contaminated. Then you have to discard the sample.</message>
```

Comment:
IIRC "translucid" is not correct English according to @gilbertohasnofb. Perhaps change it to "transparent". Also the phrase is very long and might not fit on the screen.